### PR TITLE
Add build to pull requests

### PR DESCRIPTION
**Pull request by ben-al**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Build gh-pages
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+
+# GITHUB_TOKEN permissions to allow deployment to GitHub Pages
+permissions:
```
Comment (NickWalt01) [inline .github/workflows/build.yml:9]: Do not need these permissions for the build stage

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow one concurrent deployment
+concurrency:
```
Comment (NickWalt01) [inline .github/workflows/build.yml:15]: Prob dont need this section either

File: .github/workflows/build.yml (hunk 1 above)
Comment (ben-al) [reply .github/workflows/build.yml:9]: Leaving it there, going to look at running dev deploy in the next sprint or two - causing too many issues

**Review by NickWalt01 — APPROVED** [review]

File: .github/workflows/build.yml (hunk 2 above)
Comment (ben-al) [reply .github/workflows/build.yml:15]: Same as above

**Pull request merged**